api/python/cellxgene_census/release_process.md

```diff
@@ -1,8 +1,8 @@
-# cell_census Python package release process
+# cellxgene-census Python package release process
```

Comment:
underscore or dash?   We seem to mostly use underscore....

Reply:
AFAIK, the only thing that should use the dash is the repo name.  Python packages get aliased, so we might as well use the "import" name, which is underscored